app/jobs/document_proofing_job.rb

```diff
@@ -124,6 +124,15 @@ def image_source(image_metadata)
     end
   end
 
+  def normalize_image_file(file_or_data_url)
+    return file_or_data_url if !file_or_data_url.start_with?('data:')
+
+    data_url_image = Idv::DataUrlImage.new(file_or_data_url)
+    data_url_image.read
+  rescue URI::InvalidURIError
```

Comment:
With merge of #6413, this needs to be changed:

```suggestion
  rescue Idv::DataUrlImage:: InvalidUrlFormatError
```

(You may also need to rebase / merge main for those changes to be reflected in your branch)